disable the "release" stage if forked repo (#38)
olafurpg/sbt-ci-release: readme.md

stages:
  - name: test
  - name: release

>     if: ((branch = master AND type = push) OR (tag IS present)) AND (repo = your-github-id-or-organization-id/your-repo-name)

Reviewer: ```suggestion
    if: ((branch = master AND type = push) OR (tag IS present)) AND NOT fork
```
